website/docs/docs/build/build-semantic-layer-intro.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Build your Semantic Layer"
+id: build-semantic-layer-intro
+description: "Learn about MetricFlow and build your Semantic Layer with Semantic models and Metrics"
+sidebar_label: Build your Semantic Layer
+tags: [Metrics, Semantic Layer]
+hide_table_of_contents: true
+---
+
+Build your semantic layer seamlessly using the dbt Semantic Layer, powered by MetricFlow. Use constructs like semantic models and metrics to avoid repetition and optimize your development workflow.
```

Comment:
```suggestion
Build your semantic layer seamlessly using the dbt Semantic Layer, powered by MetricFlow. Use constructs like [semantic models](/docs/build/semantic-models) and [metrics](/docs/build/metrics-overview) to avoid repetition and optimize your development workflow.
```